### DISCUSSION TITLE
Hi team i don't get git notification when issue is created. but got all  other notifications like like pull request comment on issue . please help me for this

### DISCUSSION THREAD

**Discussion by amritsingh157192** [community/community#32728] [Other Feature Feedback, Questions, & Ideas]
Hi team i don't get git notification when issue is created. but got all  other notifications like like pull request comment on issue . please help me for this

**Comment by queenofcorgis** [comment]
Hi @amitsingh157192, The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your second post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
